<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="add_button">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="record_button">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Rec</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="play_button">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pause_button">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_button">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="brush_label">
         <property name="text">
          <string>Brush:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="brush_box">
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/smallstroke.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/midstroke.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/bigstroke.png</normalon>
           </iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="color_label">
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="color_box">
         <item>
          <property name="text">
           <string>Black</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/black.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/cicon-blue.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/cicon-red.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/cicon-orange.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/cicon-yellow.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/cicon-green.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>White</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>icons/cicon-white.png</normalon>
           </iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="gv">
       <property name="cursor" stdset="0">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="progress_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="progress_label">
         <property name="text">
          <string>0:00 / 4:00</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_add"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="separator"/>
    <addaction name="action_record"/>
    <addaction name="action_pause"/>
    <addaction name="action_play"/>
    <addaction name="action_stop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_record">
   <property name="text">
    <string>Record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="action_play">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+?</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
